CONTRIBUTING.md

```diff
@@ -4,5 +4,5 @@ PyBee <3's contributions!
 
 Please be aware, PyBee operates under a Code of Conduct. 
 
-See [CONTRIBUTING to PyBee](pybee.org/contributing) for details.
+See [CONTRIBUTING to PyBee](pybee.org/contributing/index.html) for details.
```

Comment:
If both work, I'd rather use the `http://` - `index.html` is a bit of a messy URL.

BTW: How did you find this repository? I haven't made much of a noise about it yet, and I only pushed it this afternoon!